lib/Dialect/TritonGPU/IR/Dialect.cpp

```diff
@@ -285,13 +292,7 @@ SmallVector<unsigned> getOrder(Attribute layout) {
   }
   if (auto dotLayout = dyn_cast<DotOperandEncodingAttr>(layout)) {
     auto rank = getWarpsPerCTA(dotLayout.getParent()).size();
-    SmallVector<unsigned> order(rank);
-    if (isa<AMDMfmaEncodingAttr>(dotLayout.getParent())) {
-      return getOrderForDotOperand(dotLayout.getOpIdx(), rank);
-    } else {
-      std::iota(order.rbegin(), order.rend(), 0);
-    }
-    return order;
+    return getOrderForDotOperand(dotLayout.getOpIdx(), rank);
```

Comment:
This is change is correct for consistency, but will conflict with https://github.com/triton-lang/triton/pull/4891. We will have to fix there (and perhaps the LL lowering) if this one lands after the LL one.